client/me/billing-history/transactions-table.jsx

```diff
@@ -81,18 +81,26 @@ class TransactionsTable extends React.Component {
 	};
 
 	serviceNameDescription = transaction => {
+		let planName = transaction.plan;
```

Comment:
The code for prepending `Jetpack` to the plan name is pretty similar here and in the receipt page. Could we have an utility function (we already have `me/billing-history/utils`) that would take care of this? This way it could have some tests, and also we'd avoid code repetition.